### PR TITLE
Write up of current approved and waiting draft decisions

### PR DESCRIPTION
**Pull request by PonteIneptique**

### CODE REVIEW

**Comment by PonteIneptique** [comment]
There might be typos or better way to express things. This is at least an attempt to agglomerate every decision we had in a readable documentation :)

File: Document-API.md [hunk 1]
```diff
@@ -2,20 +2,167 @@
 
 The documents entry point is used to access the data for documents, as opposed to metadata (which is found in collections).  The representation of a document is up to the implementation.
 
+## Default Scheme
+
+- The document API can support as many response format as the content provider wishes.
+- The document API must support an `application/tei+xml` response format that is made of at least the following structure :
```
Comment (balmas) [inline Document-API.md]: maybe we should say "must, at minimum" ?

File: Document-API.md [hunk 2]
```diff
@@ -2,20 +2,167 @@
...
+The response contains the following response headers:
+
+| name | description |
+|------|-------------|
+| Link | Gives relation to next and previous pages |
+| Content-Type | Content type of the response (By default : `application/tei+xml`)|
+
+### URI Template
+
+Here is a template of the URI for Collection API. The route itself (`/dts/api/navigation/`) is up to the implementer.
```
Comment (balmas) [inline Document-API.md]: why does this use a different example route than the examples below (/dts/api/navigation vs /dts/api/documents)

Comment (PonteIneptique) [reply Document-API.md]: Just me making a typo... Correcting it right away.

**Comment by hcayless** [comment]
I have a question about response format: do we *have* to have a `<TEI>` element wrapper? I think it's fine, if you're just returning a fragment, to wrap the results in `<dts:fragment>` and leave it at that. The root element, whatever it is, should declare the TEI namespace, of course.

**Pull request merged**